oaknational/Oak-Web-Application
# feat(button): add brush stroke svgs for buttons

## Description

- Brush borders for button
- Brush/blob background for icon
- Fetch correct Lexend fontweight (600)
- adds `$background` prop for button/icon
- updates button `variant` prop to take values `brush` (default) and `minimal`
- adds `buttonIconBackgroundColors` section to theme. This is because a button can have the following colour areas:
  - the colour of the background of the button (passed in as `background` prop)
  - the colour of the button text (`=contrastColors[bg]`)
  - the colour of the background of the icon (if the button has an icon) (`=buttonIconBackgroundCOlors[bg]`)
  - the colour of the foreground of the icon (if the button has an icon) (`contrastColors[iconBg]`)

### Known issues (todo)
- these svgs are fetched from server, meaning there's a moment without them, before they appear. This will be fixed by #385 
- the icons themselves are still the stock icons, so they won't look like the current figma designs. This Pr is just for the background of the icon, and the borders of the button.

## Issue(s)

Fixes #365

## How to test

1. Go to storybook/app
2. Check the buttons and the icons
3. You should see them looking as below

## Screenshots

How it should now look:

![image](https://user-images.githubusercontent.com/12934669/181582839-b8f6e5e3-c56a-427a-bf8e-8f7c8486d4bb.png)

![image](https://user-images.githubusercontent.com/12934669/181582991-5a6b7730-cd98-4197-9314-5a820ddb7cf6.png)

![image](https://user-images.githubusercontent.com/12934669/181583815-58173152-fec4-4c9c-a071-c622ca8eb905.png)




## Checklist
